web/src/app/details/DetailsPage.js

```diff
@@ -30,6 +30,13 @@ const styles = theme => ({
     [theme.breakpoints.up('md')]: { float: 'left' },
     margin: 20,
   },
+  mainHeading: {
+    fontSize: '1.5rem',
+    fontFamily: '"Roboto", "Helvetica", "Arial", sans-serif',
```

Comment:
We should be good to remove this and stick with whatever default MUI gives us

Reply:
<img width="822" alt="Screen Shot 2019-07-12 at 8 23 02 AM" src="https://user-images.githubusercontent.com/29063970/61131270-6216b280-a47e-11e9-8575-cfa614543f3f.png">

These are the default styles MUI gives for an h2. I think the best approach if we want to keep the visuals the same as before and not add the CSS is to use the variant prop, and pass what was previously being rendered visually but still with the component prop for semantics.

Reply:
I will remove everything and move all this work to the headings branch I started.